README.md

```diff
@@ -75,7 +79,35 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/persons", (req, res) => {
```

Comment:
👍 This is just fine for responding to simple html forms with no support for `put` and `delete`, but for a RESTful api we would want to use the explicit `.put` and `.delete` methods provided by express:

```js
app.get('/', (req,res)=>(
    res.send('GET')
))

app.post('/', (req,res)=>(
    // res.send('POST')
    res.redirect('/POST')
))

app.put('/', (req,res)=>(
    // res.send('PUT')
    res.redirect('/PUT')
))

app.delete('/', (req,res)=>(
    // res.send('DELETE')
    res.redirect('/DELETE')
))
```